articles/howtos/make-template-decorator-ready.md

```diff
@@ -0,0 +1,120 @@
+---
+uid: MakeTemplateDecoratorReady
+---
+# How do I make a Template, Decorator Ready?
+
+Click [here](xref:Decorator) to find out more about Decorators.
```

Comment:
Similar to my comment on `create-new-decorator.md`, I wonder if it's better to put this at the end of the article as further reading so as to avoid any readers getting distracted or overwhelmed while reading this article.